src/app/core/auth-module/auth.service.ts

```diff
@@ -388,7 +388,14 @@ export class AuthService {
    *
    * @param user the user's data from the registration component
    */
-  async register(user: any): Promise<User> {
+  async register(user: {
```

Comment:
Looks good, there isn't a User DTO we could use? Just a thought

Reply:
^ there's a million places in the client that need proper typing too, would be a good story to complete later on =)

Reply:
Believe it or not, but there used to be an npm package that provided types for everything. It was removed in 2019.

Reply:
@dsoto18 There is a type in the backend, but like Andreas said, as you'll come to learn there are many refactors needed on the frontend for proper typing.

I do agree we should get to this though